docs/index.md

```diff
@@ -71,6 +71,34 @@ containerd will come with a default implementation for the various components.
 These defaults will be chosen by the maintainers of the project and should not change unless better tech for that component comes out.
 Additional implementations will not be accepted into the core repository and should be developed in a separate repository not maintained by the containerd maintainers.
 
+### Build it
+You can build `containerd` via Docker container. You can build an image from
```

Comment:
This is a roundabout way of including a Dockerfile. The build of containerd needs to be done _without_ the help of docker, such that system integrators can understand the build requirements. Just placing a dockerfile into a doc does not meet this requirement.